src/components/CompanyCard/CompanyCard.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './companyCard.scss';
+
+
+const CompanyCard = (props) => {
+
+  const lightHeadingStyle = {
+    color: 'white'
+  }
+
+  const darkHeadingStyle = {
+    color: '#e88069'
+  }
+
+  const rowReverse = {
+    flexDirection: 'row-reverse'
+  }
```

Comment:
If you want to change an element's appearance or its behavior you shouldn't focus on styles directly, but treat it as boolean params. for example props could be presented as an below object:

```
{
  title: 'component title',
  description: 'component description',
  image: <Image />,
  isLightHeading: true, // by default false
  rowReverse: true // by default false
}
```

Reply:
And with such incoming props you can introduce below logic in the `CompanyCard`:

```
const additionalHeadingStyles = {
  color: props.isLightHeading ? 'white' : 'black'
};
const additionalRowStyles = {
  flexDirection: props.rowReverse ? 'row-reverse' : 'row'
};
```

Reply:
Okay. I see what you mean. lemme try this.